src/sql/workbench/contrib/executionPlan/browser/executionPlanComparisonPropertiesView.ts

```diff
@@ -61,7 +65,8 @@ export class ExecutionPlanComparisonPropertiesView extends ExecutionPlanProperti
 			this._primaryTarget = localize('executionPlanPropertiesEdgeOperationName', "Edge");
 		}
 
-		let topTitleText = localize('executionPlanComparisonPropertiesTopOperation', "Top operation: {0}", this._primaryTarget);
+		const operationOrientation = this._orientation === ExecutionPlanCompareOrientation.Horizontal ? topOperationLabel : leftOperationLabel;
+		let topTitleText = localize('executionPlanComparisonPropertiesTopOperation', "{0}: {1}", operationOrientation, this._primaryTarget);
```

Comment:
I would suggest you create a helper function that takes in a parameter (primaryTarget) and make this a part of the strings above. e.g. https://github.com/microsoft/azuredatastudio/blob/main/extensions/arc/src/localizedConstants.ts#L81

This helps give the translators more context about how the string is going to be used and avoids any potential issues with different languages ordering stuff in different ways (for example RTL languages)